src/index.css

```diff
@@ -0,0 +1,5 @@
+* {
+  background-color: rgb(25, 184, 184);
+
+  /* color: bisque; */
+}
```

Comment:
It would be nice to remove used code for it makes you code base look untidy. It would also be nice To use the requested font face from the design requirement which are Lato ( control font ) and gill sans (body, title font)

Reply:
Resolved